jit.py

```diff
@@ -94,7 +94,7 @@ def all_to_master(self):
 
 	def pull_all(self):
 		if not self.handle_dirty_repos():
-			with multiprocessing.Pool(10) as p:
+			with multiprocessing.Pool(16) as p:
```

Comment:
Can do something like `multiprocessing.cpu_count() * 2` if you want to do something less hard-coded